python/pyspark/sql/functions.py

```diff
@@ -1592,6 +1592,57 @@ def xxhash64(*cols):
     return Column(jc)
 
 
```

Comment:
Now we should also add it into type hints and documentations at https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.sql.rst#functions and https://github.com/apache/spark/blob/master/python/pyspark/sql/functions.pyi. cc @zero323 FYI